src/components/UserSearchBar.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+
+import { Button, TextField, Typography, Link, InputBase, Paper, createStyles, Theme, withStyles } from '@material-ui/core';
+import SearchIcon from '@material-ui/icons/Search';
+interface SearchProps {
+  onSearchChange: any;
+  classes: { search: string; searchWrapper: string; inputWrapper: string; };
+}
+
+const styles = (theme: Theme) =>
+  createStyles({
+    search: {
+      color: '#828282',
+    },
+    searchWrapper: {
+      width: '100%',
+      height: '25px',
+      border: '1px solid #ff922d',
```

Comment:
According to Figma, border should be the `main` palette in `ThemeStyles.ts`  when active and otherwise `BDBDBD`. Lets put off the searchbar animation for another PR if we decide to do it since it's not that important.

Figma screenshot:
![image](https://user-images.githubusercontent.com/35501399/104109767-ed485280-5285-11eb-8d54-688818f02706.png)